### DISCUSSION TITLE
keyball61でEEPROMに設定を保存できない

### DISCUSSION THREAD

**Discussion by yukinori-kurahashi** [Yowkees/keyball#291] [Q&A]
こちらを参考に keyball61を組み立てました。
基本的にはすべての機能が正常に動作しておりますが、EEPROMの保存がうまくできません。
さらに、CPIの変更についても以下に記載のキーと設定が違うよに動作しております
WEBサイトに記載の通り、0x5DA6でEEPROMの保存ができると考えてよろしいでしょうか
また、そうだとした場合、私の環境ではEEPROMへの保存ができない原因に心当たりがありましたら教えていただけますと助かります

https://github.com/Yowkees/keyball/blob/main/keyball61/doc/rev1/buildguide_jp.md#%EF%BC%91%EF%BC%91keyball%E3%81%AE%E4%BD%BF%E3%81%84%E6%96%B9

参考にしているキーバインド
![image](https://github.com/Yowkees/keyball/assets/4408757/2bcc8413-69b8-4187-9b14-f302956e73be)

私のlayer3の設定
![image](https://github.com/Yowkees/keyball/assets/4408757/9c5d3483-a41a-471f-824e-ae0fd47b9111)

**Comment by Yowkees** [comment]
キーコードのズレについては報告をいただいておりまして、コンパイルするQMKバージョンに依存します。

最新QMKで修正したいのですが、Remapが最新QMKに対応していないため、
その他の不具合が出そうなので修正を見送っております。

特殊キーのみの不具合ですので、お手数ですが１つズレていることを念頭にキーマップをご検討いただければ幸いです。
EEPROMの保存もキーコードズレを念頭にキー入力いただけると保存できると思います。

以上、お手数ですが、ご対応お願いいたします。

**Reply by yukinori-kurahashi** [reply]
ご回答いただきありがとうございました。
特種キーが一つずれていたのですね
アドバイスいただいたやり方でやってみます。
